WebRoot/h/login

```diff
@@ -2,13 +2,6 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 
 <%-- Pass on all parameters for deeplinking --%>
-<c:url value="search" var="url">
```

Comment:
instead of removing this can't we just remove "/h/login" from web.xml

Reply:
It's not working that way

Reply:
yeah it is very tightly coupled so hard to do now